scripts/build_final_apk.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+cd $(git rev-parse --show-toplevel)
+
+./scripts/rust_build_yarn_install.sh
+yarn
+yarn build:bundle
```

Comment:
do you know what this command does exactly?
why not use:
`./gradlew assembleRelease -PsplitApk=true` ?

Reply:
I can answer that... this is the code of: `yarn build:bundle`
```
react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res && 
rimraf --glob android/app/src/main/res/drawable* && 
cd android && 
./gradlew assembleRelease -PsplitApk=true && 
cd ..
```
This is because in some version of RN don't create the JS bundle correctly, and if you build with the `gradle` only, the AKP doesn't work when install in a android device, this built needs metro to work even if you put `assembleRelease`... many people fixed this creating manually the JS Bundle and packing all the assets correctly.

Reply:
Oh wow OK this is beyond me haha thanks